include/tvm/ir/type.h

```diff
@@ -132,11 +132,10 @@ class PrimType : public Type {
    * \brief Constructor
    * \param dtype The corresponding dtype.
    */
-  TVM_DLL explicit PrimType(runtime::DataType dtype);
+  TVM_DLL explicit PrimType(runtime::DataType dtype, Span span = Span());
```

Comment:
NOTE for followup: we can send this change to main

Reply:
Agree, will do in a later PR.